.github/workflows/test-keychain.yml

```diff
@@ -0,0 +1,17 @@
+name: Test Credentials Workflow
+
+on:
+    push:
+      branches: [ main ]
+    pull_request:
+      branches: [ main ]
+      
+jobs:
+
+    cred:
+        runs-on: macos-latest
+        steps:
+        - uses: actions/checkout@v3
+
+        - name: Get test credential
+          run: ./cred/scripts/keychain_cred.sh
```

Comment:
This job might just want to run at the start of the existing MacOS CI

Reply:
Should I delete this yml file then?

Reply:
Yes, but it does look like it does what you need it to d